db/schema.rb

```diff
@@ -330,7 +329,7 @@
     t.jsonb    "preferences",         default: {}
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "roles",               default: [], null: false, array: true
+    t.jsonb    "activity_count",      default: {}
```

Comment:
Why is activity count a json field? Is there anyway it can just be an integer field, so it can be updated with `UPDATE activity_count = activity_count + 1 WHERE id = 1`? That would prevent use from having to worry about any locking issues.